notebooks/example_container_fmriprep_4review.yaml

```diff
@@ -0,0 +1,67 @@
+# This is an example config yaml file for fMRIPrep version 20.2.3.
+
+# Warning!!!
+#   This is only an example, which may not necessarily fit your purpose,
+#   or be an optimized solution for your case,
+#   or be compatible to the fMRIPrep version you're using.
+#   Therefore, please change and tailor it for your case before use it!!!
+
+# Arguments when executing the BIDS App using `singularity run`:
+# You should not have flags of : `--participant-label`, or `--bids-filter-file`!
+babs_singularity_run:
+    -w: "$BABS_TMPDIR"   # this is a placeholder. To be changed to `${PWD}/.git/tmp/wkdir`
+    --n_cpus: '1'
```

Comment:
Why have this as a string?

Reply:
Your documentation talks about putting numbers in quotes, but it's still not clear to me why this is necessary.

Reply:
Oh I was just concerned that if the number won't be presented exactly as it is in the YAML file, e.g., if `1` will be changed to float `1.0`, or if float `1.0` will be changed to int `1` when 1) YAML file reading (see below), and when 2) generating the command of `singularity run` as string. For 2), right now I'm assuming the values are strings. In short, it's just make my life easier if it's quoted as a string. Does it make sense?

I use this command to load YAML file: `config = yaml.load(f, Loader=yaml.FullLoader)`

Reply:
Just to add this rationale (eg. int/float not to be changed) to the docs